lib/web_ui/lib/src/engine.dart

```diff
@@ -190,6 +190,7 @@ export 'engine/vector_math.dart';
 export 'engine/view_embedder/dimensions_provider/custom_element_dimensions_provider.dart';
 export 'engine/view_embedder/dimensions_provider/dimensions_provider.dart';
 export 'engine/view_embedder/dimensions_provider/full_page_dimensions_provider.dart';
+export 'engine/view_embedder/display_dpr_stream.dart';
```

Comment:
This needs to be added to the license file too

Reply:
Yep, I expect this to fail the license check!